source/development/wazuh-logtest.rst

```diff
@@ -0,0 +1,63 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _dev-wazuh-logtest:
+
+
+Wazuh Logtest
+=============
+
+The Wazuh-Logtest solution is distributed by different components of the Wazuh environment:
+
+**Core Logtest**
+
+    The logtest core is part of the *Analysis daemon* (in manager). It has a session manager and shares its rules engine with Logtest.
```

Comment:
analysis daemon must be ossec-analysisd, it also solves implicit location (the deamon only runs at manager)
Lot of it and its does not exist